Client/index.js

```diff
@@ -2443,7 +2443,7 @@ app.get("/certificate", async (req, res) => {
   });
 });
 
-app.get("/codechef-iare-chapter", checkSignIn, async (req, res, next) => {
+app.get("/codechef-iare-chapter", async (req, res, next) => {
```

Comment:
```suggestion
app.get("/codechef-iare-chapter", async (req, res, next) => {
```

You are disabling user session feature for this sub url!